x/nexus/keeper/msg_server.go

```diff
@@ -167,8 +171,7 @@ func (s msgServer) activateChain(ctx sdk.Context, chain exported.Chain) {
 	}
 
 	// no chain maintainer for cosmos chains
-	if !s.axelarnet.IsCosmosChain(ctx, chain.Name) && !isActivationThresholdMet(ctx, s.Nexus, s.staking, chain) {
-		s.Logger(ctx).Info(fmt.Sprintf("activation threshold is not met for %s", chain.Name))
+	if !s.axelarnet.IsCosmosChain(ctx, chain.Name) && !s.isActivationThresholdMet(ctx, s.Nexus, s.slashing, s.snapshotter, chain) {
```

Comment:
if you are adding the msgServer as receiver, then there is no need to pass in all the keepers

Reply:
Ah, thanks, forgot to remove them